soap/src/java/com/zimbra/soap/mail/type/ActionSelector.java

```diff
@@ -153,6 +153,13 @@
     protected String tagNames;
 
     /**
+     * @zm-api-field-tag non-existent-ids
+     * @zm-api-field-description Comma-separated list of non-existent ids
+     */
+    @XmlAttribute(name=MailConstants.A_NON_EXISTENT_IDS /* nei */, required=false)
+    protected String nonExistentIds;
```

Comment:
That was a leftover of my first attempt at parsing the response.
I do agree that a boolean would be better to trigger this in the response with the default being not present.